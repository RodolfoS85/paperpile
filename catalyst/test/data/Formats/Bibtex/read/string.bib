@string{placm = "ACM Press, New York"}
@string{plspringer = "Springer, Berlin"}
@string{icdt97 = "Proceedings of the 6th International Conference on Database
Theory"}

@INPROCEEDINGS{Ullman97,
  AUTHOR = "Jeffrey D. Ullman",
  TITLE = "Information Integration Using Logical Views",
  BOOKTITLE = icdt97,
  YEAR = 1997,
  publisher = plspringer,
  }

@InProceedings{RaoRM93,
  author =   {R. Rao and D.M. Russel and J.D. Mackinlay},
  title =    {System Components for Embedded Information Retrieval
                  from Multiple Disparate Information Sources},
  booktitle =    {Proceedings of the ACM UIST '93},
  year =     1993,
  address = {Atlanta, Ga.},
  publisher = placm,
  pages =    {23--33},
}