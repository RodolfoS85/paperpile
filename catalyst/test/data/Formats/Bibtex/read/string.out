---
publisher: Springer, Berlin
year: 1997
authors: Ullman, Jeffrey D.
booktitle: Proceedings of the 6th International Conference on Database Theory
---
publisher: ACM Press, New York
authors: Rao, R. and Russel, D.M. and Mackinlay, J.D.
